viewing_party/party.py

```diff
@@ -1,23 +1,248 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}

```

Comment:
I'd suggest moving the `new_movie` declaration under the invalid data check so that we don't allocate space for `new_movie` unless we know it will be used.